features/distro-resources/src/main/resources/items/demo.items

```diff
@@ -26,77 +26,77 @@ Group:Number:AVG                Temperature "Avg. Room Temperature [%.1f °C]"
-Switch Light_FF_Office_Ceiling      "Ceiling"       (FF_Office, Lights)
-Switch Light_FF_Son_Ceiling         "Ceiling"       (FF_Son, Lights)
-Switch Light_FF_Daughter_Ceiling    "Ceiling"       (FF_Daughter, Lights)
-Switch Light_FF_Bed_Ceiling         "Ceiling"       (FF_Bed, Lights)
-
-Switch Light_C_Corridor_Ceiling     "Ceiling"       (gC, Lights)
-Switch Light_C_Staircase            "Staircase"     (gC, Lights)
-Switch Light_C_Washing_Ceiling      "Washing"       (gC, Lights)
-Switch Light_C_Workshop             "Workshop"      (gC, Lights)            ["Lighting"]
-
-Switch Light_Garden_Garage          "Garage"        (Garden, Lights)        ["Lighting"]
-Switch Light_Garden_Terrace         "Terrace"       (Garden, Lights)
+Dimmer Light_GF_Living_Table        "Table"         (GF_Living, Lights)     ["Light", "Lighting"]
+Switch Light_GF_Corridor_Ceiling    "Ceiling"       (GF_Corridor, Lights)   ["Light"]
+Switch Light_GF_Kitchen_Ceiling     "Ceiling"       (GF_Kitchen, Lights)    ["Light"]
+Switch Light_GF_Kitchen_Table       "Table"         (GF_Kitchen, Lights)    ["Light"]
+Switch Light_GF_Corridor_Wardrobe   "Wardrobe"      (GF_Corridor, Lights)   ["Light"]
+Switch Light_GF_Toilet_Ceiling      "Ceiling"       (GF_Toilet, Lights)     ["Light"]
+Switch Light_GF_Toilet_Mirror       "Mirror"        (GF_Toilet, Lights)     ["Light"]
+
+Switch Light_FF_Bath_Ceiling        "Ceiling"       (FF_Bath, Lights)       ["Light"]
+Switch Light_FF_Bath_Mirror         "Mirror"        (FF_Bath, Lights)       ["Light"]
+Switch Light_FF_Corridor_Ceiling    "Corridor"      (FF_Corridor, Lights)   ["Light"]
+Switch Light_FF_Office_Ceiling      "Ceiling"       (FF_Office, Lights)     ["Light"]
+Switch Light_FF_Son_Ceiling         "Ceiling"       (FF_Son, Lights)        ["Light"]
+Switch Light_FF_Daughter_Ceiling    "Ceiling"       (FF_Daughter, Lights)   ["Light"]
+Switch Light_FF_Bed_Ceiling         "Ceiling"       (FF_Bed, Lights)        ["Light"]
+
+Switch Light_C_Corridor_Ceiling     "Ceiling"       (gC, Lights)            ["Light"]
+Switch Light_C_Staircase            "Staircase"     (gC, Lights)            ["Light"]
+Switch Light_C_Washing_Ceiling      "Washing"       (gC, Lights)            ["Light"]
+Switch Light_C_Workshop             "Workshop"      (gC, Lights)            ["Light", "Lighting"]
+
+Switch Light_Garden_Garage          "Garage"        (Garden, Lights)        ["Light"]
+Switch Light_Garden_Terrace         "Terrace"       (Garden, Lights)        ["Light"]
 
 /* Heating */
-Switch Heating_GF_Corridor          "Corridor"      <heating>   (GF_Corridor, Heating)
-Switch Heating_GF_Toilet            "Toilet"        <heating>   (GF_Toilet, Heating)
-Switch Heating_GF_Living            "Livingroom"    <heating>   (GF_Living, Heating)
-Switch Heating_GF_Kitchen           "Kitchen"       <heating>   (GF_Kitchen, Heating)
+Switch Heating_GF_Corridor          "Corridor"      <heating>   (GF_Corridor, Heating)  ["HVAC"]
```

Comment:
That's the unresolved issue :) This is done like this only to make queries return the correct results, and there is nothing implemented so far (e.g. in the SemanticsService) to enforce a group/member relationship between an Equipment and its Points. So those are currently Equipments with no Points...

Reply:
My suggestion [here](https://github.com/eclipse/smarthome/pull/6288#issuecomment-427625011) was to allow Equipment tags next to Point tags in such situations - but imho we should not leave out the Point tag unless we again define a derived default.
As "HVAC" can be all sort of stuff, I doubt we can derive a generally valid default, hence I think we should add "Control" & "Temperature" (or, as discussed for the Lights, only add "Temperature" and have "Control" being derived. Wdyt?

Reply:
I guess it kinds of makes sense semantically to consider a radiator or HVAC equipment as a way to control the temperature, but it doesn't help at all for the use case of giving items general names by which they can be referred to. In this case this will even be detrimental since it will mix HVAC/radiators controls with temperature measurements in queries about the "temperature" property, which is not what the user would expect, and will probably have other side effects. I think Points are rarely referred to directly, mostly Equipments, Properties and Locations.

So unless we find a way to:
> allow Equipment tags next to Point tags in such situations

then I see no solution apart to acknowledge that the semantic tags aren't always a reliable source of terms to address the items, and give a way to override them altogether as a source, e.g. using some private metadata, which would provide no additional semantic value to the graph but will supply the correct terms (example: `Switch Heating_GF_Corridor ["Temperature", "Control"] { terms="HVAC,RadiatorControl"`). This is quite easily feasible, but obviously not ideal because it defeats the whole purpose of giving semantics to items in this case. I would very much prefer the "equipment with a point" idea to materialize.

Reply:
> But you are referring to them in a way that you expect a certain functionality from them - and the functionality corresponds to the Point tag. A Radiator can potentially have many different Points. If there is only one, we must have some implicit understanding what this is. Thinking about it, it is really not that much "controlling temperature", but switching its power on/off.

> Yes, but the items correspond to Points and they carry this implicitly - what I am trying is to derive from these implicit assumptions an explicit definition.

Yes, I realize that, but if there is only one item for the equipment, then imho it doesn't really matter in terms of functionality what kind of Point it is, it can even in most cases be derived from the type of item alone. For instance, "turn on the radiator in the kitchen" if there's only one matching item and it's a Switch, or "open the blinds" and there's only Rollershutter items matching, then it's pretty obvious what to do, I believe there's no real need to tag those explicitely as Points.
In other words, we can almost assume all Switches are "Switch" points unless tagged differently like "Tampered" or "LowBattery", and Rollershutters are "OpenState" ("OpenLevel"?) points.

> Well, a query for "temperature measurements" should anyhow not include "temperature controls", should it?

currently the skill responding to "[give me the] X in the Y (location)" means "give me all you have for X in Y", which includes measurements, controls, etc. An user would be probably surprised to get HVAC switches when asking for the temperature.

> Note that my suggestion was "to allow Equipment tags next to Point tags" - i.e. do not have to introduce "terms", but you have the Equipment synonyms associated with it.

That's what I understood too, and is clearly my preference as well.

Reply:
>  it can even in most cases be derived from the type of item alone

That was my question: If it can be derived, we should define what it is.

> I believe there's no real need to tag those explicitly as Points.

Just what I am saying. As long as we can derive something, there is no need for an explicit tag. But if we cannot, we should require a Point tag.

> all Switches are "Switch" points

Brings us to the interesting question of what a "Switch Point" exactly is. It is a subtype of "Control" and Control (like Measurement) should have a Property associated with it. "Power" would seem to refer to some external power, while we want something like "Power of the Equipment". Maybe a "Status Point" might actually the better match? 

> An user would be probably surprised to get HVAC switches when asking for the temperature.

Really? If you reformulate it to "give me all that is related to temperature", it should imho be indeed included. I think besides X and Y, we will have to get some Z out of the questions in future as you in fact often refer to some functionality Z of X in Y, like "battery level of X", "turn on X", "switch X to mode Z", etc. But no worries, I don't expect this by Sunday ;-)

Reply:
Then I wouldn't be shocked if an item tagged as an Equipment would represent both the Equipment and the Point describing its main functionality (derived from its type) - if it's not a group or tagged with a Point tag -, i.e. the implementation of `SemanticsPredicates.isA` would take that into account and `SemanticsPredicates.isA(HVAC.class)` and `SemanticsPredicates.isA(Switch.class)` would both yield true in this case.

Now how to represent that in the item's metadata is another story, but ultimately item metadata is probably not the best way to publish semantics, since in the future Equipments might also be Things (or "virtual"), Points may be trigger Channels, Locations could also be derived from the location of Things etc.
So item metadata would be too restricted and there would be a need for a "real" domain model and associated API, maybe even including REST endpoints like
`GET /rest/semantics/equipments/{equipmentId}/points` or
`GET /rest/semantics/points/{pointId}/location` and so on.
The equipmentId or pointId could refer either only to an Item, or another ESH entity like a Thing or Channel, or nothing at all.